azure-resources/Web/sites/recommendations.yaml

```diff
@@ -113,3 +113,63 @@
   learnMoreLink:
     - name: Configure web apps in Azure App Service
       url: "https://learn.microsoft.com/azure/app-service-web/web-sites-configure"
+
+- description: Health check
+  aprlGuid: fd049c28-ae6d-48f0-a641-cc3ba1a3fe1d
+  recommendationTypeId: null
+  recommendationControl: Other Best Practices
+  recommendationImpact: Medium
+  recommendationResourceType: Microsoft.Web/sites
+  recommendationMetadataState: Active
+  longDescription: |
+    Use Health Check for production workloads. Health check increases your application's availability by rerouting requests away from unhealthy instances, and replacing instances if they remain unhealthy. Your App Service plan should be scaled to two or more instances to fully utilize Health check. The Health check path should check critical components of your application. For example, if your application depends on a database and a messaging system, the Health check endpoint should connect to those components. If the application can't connect to a critical component, then the path should return a 500-level response code to indicate the app is unhealthy. Also, if the path does not return a response within 1 minute the health check ping is considered unhealthy.
```

Comment:
longDescription has a limit of 300 characters, including spaces.

Reply:
Updated to be within 300 characters.